packages/sw-offline-google-analytics/test/unit/replay-queued-requests.js

```diff
@@ -0,0 +1,132 @@
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+/* eslint-env mocha, browser */
+/* global chai, goog, fetch, MockDate */
+
+'use strict';
+
+describe('replay-queued-requests', () => {
+  const constants = goog.offlineGoogleAnalytics.test.constants;
+  const enqueueRequest = goog.offlineGoogleAnalytics.test.enqueueRequest;
+  const replayRequests = goog.offlineGoogleAnalytics.test.replayRequests;
+  const IDBHelper = goog.offlineGoogleAnalytics.test.IDBHelper;
+
+  const idbHelper = new IDBHelper(constants.IDB.NAME, constants.IDB.VERSION,
+     constants.IDB.STORE);
+
+  let fetchedUrls = [];
+  const urlPrefix = 'https://replay-queued-requests.com/';
+  // An arbitrary, but valid, timestamp in milliseconds.
+  const initialTimestamp = 1470405670000;
+  // A 1000 millisecond offset.
+  const timestampOffset = 1000;
+  const originalFetch = window.fetch;
+
+  beforeEach(function() {
+    MockDate.set(initialTimestamp + timestampOffset);
+    fetchedUrls = [];
+    fetch = (requestUrl) => {
```

Comment:
Curious about the move from the mocked `fetch` from npm to this brute-force mocking. I'm pretty sure it's fine, but I wonder whether it was motivated by an incompatibility with the new build setup, or something else.